lib.py

```diff
@@ -0,0 +1,309 @@
+    time.sleep(10)
+    bot.send_message(m.chat.id, 'Never gonna give you up!')
+    time.sleep(2)
+    bot.send_message(m.chat.id, 'Never gonna let you down!')
+    time.sleep(2)
+    bot.send_message(m.chat.id, 'Never gonna run arooooooouuuund and desert you!😆))')
+
+
+def info(bot, m):
+    """
+    INFO command processing
+    Sends the message (authors info) to the User
+    """
+    author1 = '[ML777](tg://user?id=107257538)'
+    author2 = '[axxeny](tg://user?id=000492785)'
+    bot.send_message(m.chat.id, f'Bot made by {author1} and {author2}',
+                     parse_mode='MarkdownV2',
+                     disable_web_page_preview=True,
+                     )
+
+
+def feedback(bot, m):
+    """
+    INFO command processing
+    Sends the message (link to Slack English Club Channel) to the User
+    """
+    hyper_text = '[\#english\-club CMF Slack channel\!](https://cmf2021.slack.com/archives/C02D8FP2CPQ)'
+    bot.send_message(m.chat.id,
+                     "You can write your feedback at {}".format(hyper_text),
+                     parse_mode='MarkdownV2',
+                     disable_web_page_preview=True)
+
+
+def link(user):
+    return f'@{user}'
+
+
+def round_create(bot, m):
+    """
+    Create round every 3 days
+    Send messages to each person in a pair at the beginning of the round
+    """
+    #while True:
+    # time.sleep(24*60*60)
+    #time.sleep(10)
+    users = read_users()
+    # посчитать пары ()
+    print('>>>round_create start')
+    print('here are the users:')
+    print(users)
+    # users_l = list(map(list, users.items()))
+    # print('here are the users_l:')
+    # print(users_l)
+    # random.shuffle(users_l)
+    # print('here are the users_l_rand:')
+    # print(users_l)
+    # it = iter(users_l)
+
+    user_list = list(users.keys())
+    random.shuffle(user_list)
+    user_shuffle_dict = {user: users[user] for user in user_list}
+    users = user_shuffle_dict
+
+    print('here are the users_rand:')
+    print(users)
+    it = iter(users)
+
+    pairs = list(zip_longest(it, it))
+    print('here are the pairs:')
+    print(pairs)
+    if pairs[-1][-1] is None:
+        pairs[-2] += (pairs[-1][0],)
+        pairs.pop()
+    print('here are the new pairs:')
+    print(pairs)
+    print(type(pairs))
+    print('for:')
+    for pair in pairs:
+        print(pair)
+        print(type(pair))
+    # для каждой пары отправить сообщения
+    for pair in pairs:
+        for user in pair:
+            partners = set(pair) - {user}
+            bros = [
+                '[' + str(bro) + '](tg://user?id=' + str(users[bro]) + ')'
+                for bro in partners
+            ]
+            bot.send_message(users[user],
+                             f'Hi, {user}\! {" , ".join(bros)} is waiting for you to speak\!\n'
+                             'Please\, contact your chat partner and plan your call within the next 3 days\.\n'
+                             'Have a nice meet\!',
```

Comment:
Перед точкой и восклицательным знаком не нужен `\`
```suggestion
                             f'Hi, {user}! {" , ".join(bros)} is waiting for you to speak!\n'
                             'Please, contact your chat partner and plan your call within the next 3 days.\n'
                             'Have a nice meet!',
```